manim/mobject/opengl_mobject.py

```diff
@@ -44,6 +45,13 @@ class OpenGLMobject:
         ("point", np.float32, (3,)),
     ]
     shader_folder = ""
+    points = _Data()
```

Comment:
Why are the class attributes needed as well as the instance ones?

Reply:
From my understanding, the class variable here tells manim that this is a `_Data` variable and defines how it will later be accessed/ defined. The instance variable in `init_data`  relies on this definition and sets the variable within `self.data` of the particular instance of that mobject i.e. (`Square()`). It might(??) be *technically* possible to only use an instance/class variable? But this is the convention I've seen for descriptors after researching it.

Here's also a guide from the python docs that shows helpful examples:
https://docs.python.org/3/howto/descriptor.html#simple-example-a-descriptor-that-returns-a-constant

https://docs.python.org/3/howto/descriptor.html#managed-attributes

Reply:
Ah I see, it makes sense now and the implementation looks good in that case